backend/main.py

```
@@ -1,468 +1,163 @@
+        "health": "/api/health"
+    }
 
 
+# Development server
 if __name__ == "__main__":
-    port = int(os.environ.get("PORT", 8000))
-    uvicorn.run(app, host="0.0.0.0", port=port)
\ No newline at end of file
+    uvicorn.run(
+        "main_new:app",
+        host="0.0.0.0",
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)